obspy/io/nordic/ellipse.py

```diff
@@ -150,6 +171,29 @@ def __repr__(self):
         s += ')'
         return s
 
+    def __eq__(self, other):
+        """
+        Returns true if two Ellipses are equal
+
+        :param other: second Ellipse
+        :type other:  :class: `~ellipsoid.Ellipse`
+        :return: equal
+        :rtype: bool
+        """
+        eps = 1e-5
```

Comment:
It might be worth having this as an `rtol` attribute on the `Ellipse` object so that it can be accessed later if needed.

Reply:
OK, I did this.